src/main/java/opendcs/dao/SiteDAO.java

```diff
@@ -290,26 +300,27 @@ public void fillCache()
 	{
 		debug3("(Generic)SiteDAO.fillCache()");
 
-		HashMap<DbKey, Site> siteHash = new HashMap<DbKey, Site>();
+		final HashMap<DbKey, Site> siteHash = new HashMap<DbKey, Site>();
+		String q = "SELECT " + siteAttributes + " FROM " + siteTableName;
 //		ArrayList<Site> siteList = new ArrayList<Site>();
```

Comment:
This particular line isn't using bind variables because it's the entire table. there's no conditions.
This change is just to the new handlers so the connection is properly wrapped.

Reply:
But also yes, for any query that is using "where ... in (items)" I haven't figured out a simple way to handle that.
Temp tables are a good candidate but there's a different in setup between Oracle and Postgres. And none of the drivers let you just bind an array.

I've done things with anonymous procedures that would be similar in both but the syntax is different. Not insurmountable but doesn't lend itself to embedding the query. SiteDao technically needs to support Oracle and Postgres as we have that available to users.